packages/plugin-devtools/test/devtools.test.ts

```diff
@@ -0,0 +1,735 @@
+            const button = devtools.querySelector('devtools-navbar button');
+            expect(button.textContent).to.equal('Inspector');
+            expect(button.classList.contains('selected')).to.equal(true, 'button is selected');
+            const panel = devtools.querySelector('devtools-panel.inspector.active');
+            expect(!!panel).to.equal(true, 'panel is active');
+        });
+        it('should resize the devTools', async () => {
+            const navbar = devtools.querySelector('devtools-navbar');
+            const pos = navbar.getBoundingClientRect();
+            const begin = { clientX: pos.left + 5, clientY: pos.top + 5, bubbles: true };
+            const step = { clientX: pos.left + 15, clientY: pos.top + 25, bubbles: true };
+            const end = { clientX: pos.left + 30, clientY: pos.top - 50, bubbles: true };
+            navbar.dispatchEvent(new MouseEvent('mousedown', begin));
+            await longNextTick();
+            window.dispatchEvent(new MouseEvent('mousemove', step));
+            await longNextTick();
+            window.dispatchEvent(new MouseEvent('mousemove', end));
+            await longNextTick();
+            navbar.dispatchEvent(new MouseEvent('mouseup', end));
+            await nextTick();
+            navbar.dispatchEvent(new MouseEvent('click', end));
+            await nextTick();
+            expect(devtools.classList.contains('closed')).to.equal(false);
+        });
+        it('should close the devTools', async () => {
+            const navbar = devtools.querySelector('devtools-navbar');
+            navbar.dispatchEvent(new MouseEvent('mousedown', { bubbles: true }));
+            await nextTick();
+            navbar.dispatchEvent(new MouseEvent('mouseup', { bubbles: true }));
+            await nextTick();
+            navbar.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+            await nextTick();
+            expect(devtools.classList.contains('closed')).to.equal(true);
+        });
+        it('should open the devTools when click on button "Commands"', async () => {
+            const button = devtools.querySelector('devtools-navbar button:nth-child(2)');
+            button.dispatchEvent(new MouseEvent('mousedown', { bubbles: true }));
+            await nextTick();
+            button.dispatchEvent(new MouseEvent('mouseup', { bubbles: true }));
+            await nextTick();
+            button.dispatchEvent(new MouseEvent('click', { bubbles: true }));
+            await longNextTick();
+            expect(devtools.classList.contains('closed')).to.equal(false);
+        });
+    });
+    describe('InspectorComponent', () => {
+        it('should change panel to "Inspector" with top button', async () => {
+            const button = devtools.querySelector('devtools-navbar button');
+            expect(button.textContent).to.equal('Inspector');
+            expect(button.classList.contains('selected')).to.equal(false, 'button is not selected');
```

Comment:
Wait, how is it false ? This is another describe, it should not depend on previous tests !